docs/pages/operations-guide.md

```diff
@@ -851,9 +851,54 @@ $ curl -k -X POST -H "Authorization: token=$(dcos config show core.dcos_acs_toke
 }
 ```
 
+## Stop a pod
+
+Stopping a pod relaunches it in an idle command state. This allows the operator to debug the contents of the pod, possibly making changes to fix problems. While these problems are often fixed by just replacing the pod, there may be cases where an in-place repair or other operation is needed. For example:
```

Comment:
Not sure about the formatting, but we could start "For example:" as a new paragraph here.

Reply:
Added a paragraph break in #1771